### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by raspopov** [hufrea/byedpi#61] [Ideas]
Maybe it's time to choose a program logo? For example, ByeDPIAndroid uses an image of a dove. 

Может быть пришло время выбрать логотип программы? Например, ByeDPIAndroid использует изображение голубя. 

[logo.svg](https://github.com/dovecoteescapee/ByeDPIAndroid/blob/master/.github/images/logo.svg)

**Comment by lufog** [comment]
То скорей аватарка автора, а не логотип. Лично я, не вижу особого смысла в логотипе чисто консольной программы.
